string src
string term_id
---
memory/TermMetadata meta
